tests/net/mod.rs

```diff
@@ -318,6 +343,9 @@ where
     message_limit: Option<usize>,
     /// Optional time limit.
     time_limit: Option<time::Duration>,
+    /// Property to cause an error if a `Fault` is output from a correct node. By default,
+    /// encountering a fault leads to an error.
+    error_on_fault: bool,
```

Comment:
Do we ever want to turn that off?
I imagine that we'll always want any tests to fail if a correct node blames a correct node?

Reply:
There was a comment asking to have this option but I cannot find it now.

Reply:
By disabling errors on faults you would be able to see more faults and get stalling or divergence.

Reply:
Not sure if that's really useful, but I'm fine with it, too.